src/main/java/net/snowflake/ingest/streaming/SnowflakeStreamingIngestChannel.java

```diff
@@ -271,6 +288,8 @@ InsertValidationResponse insertRows(
   InsertValidationResponse insertRows(
       Iterable<Map<String, Object>> rows, @Nullable String offsetToken);
```

Comment:
we might need to add something like

 insertRows(Iterable<Map<String, Object>> rows,  List<String> offsetTokens)
 
 for partial ingestion

Reply:
Yes, that's the one below, offset_token will be part of the row now.